//Define the version of proto
syntax = "proto3";


message Input {
  string input_message = 1;
}

message OutputMessage {
  string output_message = 1;
}


service Predict {
  rpc EnergyConsumption(Input) returns (OutputMessage);
}
